services/docker-compose.yml

```
@@ -393,6 +393,12 @@ services:
       - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_CHECK_ENABLED}
       - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_INTERVAL_SEC=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_INTERVAL_SEC}
       - RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_COUNTER_FAIL=${RESOURCE_USAGE_TRACKER_MISSED_HEARTBEAT_COUNTER_FAIL}
+      - RESOURCE_USAGE_TRACKER_S3=${RESOURCE_USAGE_TRACKER_S3}
+      - S3_ACCESS_KEY=${S3_EXPORT_ACCESS_KEY}
+      - S3_BUCKET_NAME=${S3_EXPORT_BUCKET_NAME}
+      - S3_ENDPOINT=${S3_EXPORT_ENDPOINT}
+      - S3_SECRET_KEY=${S3_EXPORT_SECRET_KEY}
+      - S3_SECURE=${S3_EXPORT_SECURE}
```

Please check the quotes, but something like this should work for you and it also allows us to more cleanly specify if the setting is enabled/disabled. Also you can now also have assess to the opsarc-s3 if you ever require it.

```suggestion
      - RESOURCE_USAGE_TRACKER_S3='{"S3_ACCESS_KEY":"${S3_EXPORT_ACCESS_KEY}", "S3_BUCKET_NAME": "${S3_EXPORT_BUCKET_NAME}", "S3_ENDPOINT": "${S3_EXPORT_ENDPOINT}", "S3_SECRET_KEY": "${S3_EXPORT_SECRET_KEY}","S3_SECURE": "${S3_EXPORT_SECURE}"}'
```

---

Now that I think about it, this should ideally go into the devops env vars config.
Here it might just be `RESOURCE_USAGE_TRACKER_S3=${RESOURCE_USAGE_TRACKER_S3}`.


I was thinking that if you are trying to reuse the `S3Settings`, there is no clean way of doing it. You can still make mistakes and use the simcore s3 bucket in place of the one you intend using.

---

Ok i will see what is the best :+1: